init/eessi_environment_variables

```diff
@@ -0,0 +1,67 @@
+# Allow for a silent mode
+if [[ -v EESSI_SILENT ]]; then
+  # EESSI_SILENT set
+  output=/dev/null
+else
+  output=/dev/stdout
+fi
+
+export EESSI_PILOT_VERSION="2020.12"
+export EESSI_PREFIX=/cvmfs/pilot.eessi-hpc.org/$EESSI_PILOT_VERSION
+
+if [ -d $EESSI_PREFIX ]; then
+  echo "Found EESSI pilot repo @ $EESSI_PREFIX!" >> $output
+
+  # Set EPREFIX since that is basically a standard in Gentoo Prefix
+  export EPREFIX=$EESSI_PREFIX/compat/$(uname -m)
```

Comment:
@ocaisa This needs to be tweaked because there's an extra subdirectory `linux/` now, see https://github.com/ocaisa/software-layer/pull/1